File: 1217. Minimum Cost to Move Chips to The Same Position/solutions.ts
function minCostToMoveChips(position: number[]): number {
    const d = [0, 0]
    for(let i = 0; i < position.length; i++) {
        d[position[i]%2] += 1
    }
    return Math.min(d[0], d[1])
};

File: 1217. Minimum Cost to Move Chips to The Same Position/solutions.py
class Solution(object):
    def minCostToMoveChips(self, position):
        """
        :type position: List[int]
        :rtype: int
        """
        d = {0:0, 1:0}
        for p in position:
            d[p%2]+=1
        
        return min(d[0], d[1])
